rest/changestest/changes_api_test.go

```diff
@@ -2063,9 +2062,14 @@ func TestChangesIncludeDocs(t *testing.T) {
 	// Flush the rev cache, and issue changes again to ensure successful handling for rev cache misses
 	testDB.GetSingleDatabaseCollection().FlushRevisionCacheForTest()
 	// Also nuke temporary revision backup of doc_pruned.  Validates that the body for the pruned revision is generated correctly when no longer resident in the rev cache
-	assert.NoError(t, testDB.Bucket.DefaultDataStore().Delete(base.RevPrefix+"doc_pruned:34:2-5afcb73bd3eb50615470e3ba54b80f00"))
+	dataStores, _ := testDB.Bucket.ListDataStores()
```

Comment:
As mentioned below, getting the datastore from the DatabaseCollection would be simpler here, instead of trying to find it in the bucket.